dev_info/build.html

```diff
@@ -0,0 +1,31 @@
+﻿<!DOCTYPE html>
+
+<link rel="stylesheet" href="../locales/help_enu/styles.css" type="text/css" />
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta charset="utf-8" />
+<title>Building GEDKeeper</title>
+</head>
+<body>
+<h1>How to build GEDKeeper</h1>
+<h2>Microsoft Windows</h2>
+<h3>Dependencies</h3>
+<p>The following is a list of required software to build and test:
+<ul>
+<li>Microsoft <a href="https://dotnet.microsoft.com/">.NET Framework</a> 4.0.30319</li>
+<li>Microsoft <a href="https://visualstudio.microsoft.com/">Visual Studio</a> 2017 or later</li>
+<li>Nullsoft Scriptable Install System (<a href="https://sourceforge.net/projects/nsis/">NSIS</a>) 3.04</li>
+<li><a href="https://sourceforge.net/projects/sevenzip/">7-zip</a></li>
+<li><a href="https://github.com/nunit">NUnit</a> 2.6.4 (unit testing framework for .NET)</li>
+</ul>
+</p>
+<h3>Execution</h3>
+<p>Run the executable <kbd>build_all_mswin.x86.cmd</kbd> from the command line.</p>
+<p>Optionally use the parameter &quot;Release&quot; for a release build (the default is a debug build). Testing is not done with a release build.</p>
+
+<h2>Linux</h2>
+<p>Sorry, not yet implemented.</p>
+<p>Make it!</p>
```

Comment:
This shouldn't be inside a build manual.  :-)

Reply:
Yes

Reply:
I agree, but it was in the original so I left it. Not sure if @Serg-Norseman is saying "yes" to he agrees with you or "yes" it belongs, so I left it.

Reply:
Oh, that's my fault.  I wrote build.txt long ago.  I don't event remember why I wrote those lines (because (GNU) make was/is unsupported may be?).  Therefore, I would remove both \`\`Sorry, not yet implemented'' and \`\`Make it!'' from build.txt and build.html.